SwitchSpeak-iOS/SwitchSpeak-iOS/TouchGrid/ButtonNode.swift

```diff
@@ -14,21 +14,32 @@ import UIKit
 */
 class ButtonNode: Node {
 	var button = UIButton(type: UIButtonType.custom)
+    var dummy = false
 	
 	init(button: UIButton) {
 		self.button = button
 		super.init()
 	}
 	
 	override func highlightSubTree() {
+        if dummy==true{
+            return
+        }
 		let uhcolor = UIColor(red: 200.0/255.0, green: 100.0/255.0, blue: 100.0/255.0, alpha: 1.0)
 		button.layer.borderColor = uhcolor.cgColor
 	}
 	
 	override func unHighlightSubTree() {
+        if dummy==true{
+            return
+        }
 		let uhcolor = UIColor(red: 100.0/255.0, green: 130.0/255.0, blue: 230.0/255.0, alpha: 1.0)
 		button.layer.borderColor = uhcolor.cgColor
 	}
+   
+    func ifdummy ()->Bool{
```

Comment:
This should probably be named `isDummy`, since it just returns the value of dummy. `ifdummy` could imply that there is some control structure happening here.